### ISSUE TITLE
Add Tensor-RT integration

### ISSUE THREAD

**Issue by rlouf** [outlines-dev/outlines#736]
Following #728 we could add a Tensor-RT integration that would allow to use outlines's constrained generation via logits processors.

**Comment by cpfiffer** [comment]
I don't think anyone is working on this. If you are interested, you'd have to do this in the v1.0 branch to avoid having to overhaul it when we release 1.0. We're currently writing the docs at the moment, which might help with implementation.

**Comment by rlouf** [comment]
No one is. I would integrate `outlines-core` directly instead of going through outlines.

**Comment by kernel-loophole** [comment]
this is great will start working on this soon ,any starting point or arc review for this
